Assets/BossRoom/Scripts/Shared/Net/UnityServices/Lobbies/LobbyAPIInterface.cs

```diff
@@ -115,7 +115,15 @@ public async Task<Lobby> QuickJoinLobby(string requesterUasId, Dictionary<string
 
         public async Task RemovePlayerFromLobby(string requesterUasId, string lobbyId)
         {
-           await ExceptionHandling(Lobbies.Instance.RemovePlayerAsync(lobbyId, requesterUasId));
+            try
+            {
+                await ExceptionHandling(Lobbies.Instance.RemovePlayerAsync(lobbyId, requesterUasId));
+            }
+            catch (Exception e)

```

Comment:
You can probably just catch LobbyServiceException specifically here, instead of generic Exception, then do the reason check in the filter

Reply:
Currently, this message is only subscribed to by the utility that creates popup messages, and it already ignores PlayerNotFound, so it is not necessary, but we might want later to have something else subscribe to it and do something with this specific exception